core/shared/src/main/scala/scalaz/zio/ZManaged.scala

```diff
@@ -465,14 +465,15 @@ final case class ZManaged[-R, +E, +A](reserve: ZIO[R, E, Reservation[R, E, A]])
                 )
             }, {
               case (_, leftFiber) =>
-                leftFiber.interrupt
```

Comment:
I am not sure about this one.
The way it is right now this would interrupt the reservation step of Managed, which is inconsistent with the rest of the api. Furthermore await the result of the reservation could lead to unintuitive behavior with it is uninterruptible.

I believe that this is a nicer way to handle this. @jdegoes what is your opinion on this?

Reply:
Reservation should never be interrupted, and by design, is generally not interruptible, anyway. So yes, we should `await`. 👍